migrations/V1__baseline_schema.sql

```diff
@@ -0,0 +1,300 @@
+-- DROP existing tables and views
+DROP VIEW if exists kpi_values;
+DROP TABLE if exists kpi_values_history;
+DROP TABLE if exists kpi_values;
+DROP TABLE if exists target;
+DROP TABLE if exists circle_kpi_definition;
+DROP TABLE if exists kpi_definition;
+DROP TABLE if exists circle_user;
+DROP TABLE if exists kpi_user;
+DROP TABLE if exists circle;
+
+-- DROP all existing types
+DROP TYPE IF EXISTS role;
+DROP TYPE IF EXISTS action;
+DROP TYPE IF EXISTS periodicity;
+DROP TYPE IF EXISTS unit;
+
+-- Define types
+CREATE type periodicity as enum ('daily', 'weekly', 'monthly', 'quarterly', 'yearly');
+CREATE type unit as enum ('%', 'boolean', 'numeric');
+CREATE type role as enum ('admin', 'user');
+CREATE type action as enum ('CREATE', 'UPDATE', 'DELETE');
+
+-- Enable MODDATETIME extension
+CREATE extension if not exists moddatetime schema extensions;
+
+CREATE TABLE public.kpi_definition (
+    kpi_id int8 generated by default as identity,
+    kpi_name varchar(255) not null unique,
+    description varchar(255),
+    periodicity periodicity not null,
+    value_min int8,
+    value_max int8,
+    unit unit,
+    created_at timestamp default current_timestamp,
```

Comment:
Just as I mentioned in the other PR (which btw we might close now right?) I think this should be not null since we use this column for comparisons later.